Bottom: 0da3d4cd5116f81074167781aaf591f91c5f452e
Top:    0d8e643f70fd4fd538b76f6a1a7adf00fd8f446a
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 09:17:44 -0500

Add new dependency, bc, to Dockerfile


---
diff --git a/docker/buildbot-worker-ptxdist/Dockerfile b/docker/buildbot-worker-ptxdist/Dockerfile
index ee6bfeb5..b14d49c4 100644
--- a/docker/buildbot-worker-ptxdist/Dockerfile
+++ b/docker/buildbot-worker-ptxdist/Dockerfile
@@ -22,6 +22,7 @@ RUN apt-get update \
 &&  DEBIAN_FRONTEND=noninteractive \
     apt-get -y install -q \
         autoconf \
+        bc \
         bison \
         build-essential \
         ccache \
